yast/package/yast2-smt.spec

```diff
@@ -44,8 +44,8 @@ Requires:	/usr/bin/getent
 Requires:       sudo
 # Modified smt-catalogs (added --batch-mode)
 # SMT::Client::getPatchStatusLabel returning two values
-# require smt version having SMT::Curl
-Requires:	smt >= 1.1.23
+# don't require SMT (instead ask to install it), but prevent older versions
+Conflicts:	smt < 1.1.23
```

Comment:
Well, since current version is 3.0.2, I will add conflict to anything older, that should be pretty safe.